pygmt/tests/test_meca.py

```diff
@@ -8,24 +8,73 @@
 from pygmt.helpers import GMTTempFile
 
 
-@pytest.mark.mpl_image_compare
-def test_meca_spec_dictionary():
+@pytest.mark.mpl_image_compare(filename="test_meca_spec_single_focalmecha.png")
+@pytest.mark.parametrize("inputtype", ["dict_mecha", "dict_full", "array1d", "pandas"])
+def test_meca_spec_single_focalmecha(inputtype):
+    """
+    Test passing a single focal mechanism to the spec parameter.
+    """
```

Comment:
Different `inputtype` are different ways to passing a single focal mechanism to the `spec` parameter.